docs_source_files/content/findAndReplaceText.sh

```diff
@@ -0,0 +1,32 @@
+# find and copy:
```

Comment:
What is this file? Did it get committed by mistake?

Reply:
sorry. was a script to create all the de files, I removed it already